<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProfilesDialog</class>
 <widget class="QDialog" name="ProfilesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>631</width>
    <height>356</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="mainLayout">
     <item>
      <layout class="QVBoxLayout" name="profilesLayout" stretch="0,1,0">
       <item>
        <widget class="QLabel" name="profilesLabel">
         <property name="text">
          <string>List of profiles:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListView" name="profilesListView">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Ignored" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="profilesActionsLayout" stretch="0,0">
         <item>
          <widget class="QPushButton" name="addButton">
           <property name="text">
            <string>Add</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="deleteButton">
           <property name="text">
            <string>Delete</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QFormLayout" name="formLayout">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="profileLabel">
         <property name="text">
          <string>Profile:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="profileIdLabel">
         <property name="text">
          <string notr="true">0</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="nameLabel">
         <property name="text">
          <string>Name:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="nameEdit"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="serverLabel">
         <property name="text">
          <string>Server:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="serverComboBox">
         <property name="currentText">
          <string notr="true">Atys</string>
         </property>
         <item>
          <property name="text">
           <string>Atys</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Yubo</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="executableLabel">
         <property name="text">
          <string>Executable:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <layout class="QHBoxLayout" name="executableLayout" stretch="1,0,0">
         <item>
          <widget class="QLabel" name="executablePathLabel">
           <property name="text">
            <string notr="true">ryzom_client_r.exe</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="executableDefaultButton">
           <property name="text">
            <string>Default</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="executableBrowseButton">
           <property name="text">
            <string>Browse...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="argumentsLabel">
         <property name="text">
          <string>Arguments:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="argumentsEdit"/>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="commentsLabel">
         <property name="text">
          <string>Comments:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QPlainTextEdit" name="commentsEdit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>1</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="directoryLabel">
         <property name="text">
          <string>Directory:</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0">
         <item>
          <widget class="QLabel" name="directoryPathLabel">
           <property name="text">
            <string notr="true">~/.ryzom/0</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="directoryButton">
           <property name="text">
            <string>Open</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="langLabel">
         <property name="text">
          <string>Language:</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="shortcutsLabel">
         <property name="text">
          <string>Create shortcuts:</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <layout class="QVBoxLayout" name="shortcutsLayout">
         <item>
          <widget class="QCheckBox" name="desktopShortcutCheckBox">
           <property name="text">
            <string>Desktop</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="menuShortcutCheckBox">
           <property name="text">
            <string>Start Menu</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="7" column="1">
        <widget class="QComboBox" name="langComboBox">
         <item>
          <property name="text">
           <string>English</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Francais</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Deutsch</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Español</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Russian (РУССКИЙ)</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>profilesListView</tabstop>
  <tabstop>addButton</tabstop>
  <tabstop>deleteButton</tabstop>
  <tabstop>nameEdit</tabstop>
  <tabstop>serverComboBox</tabstop>
  <tabstop>executableBrowseButton</tabstop>
  <tabstop>argumentsEdit</tabstop>
  <tabstop>directoryButton</tabstop>
  <tabstop>langComboBox</tabstop>
  <tabstop>desktopShortcutCheckBox</tabstop>
  <tabstop>menuShortcutCheckBox</tabstop>
  <tabstop>commentsEdit</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ProfilesDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ProfilesDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
